examples/promptAPerson.yml

```
@@ -3,13 +3,30 @@ data-silos:
     description: Notify backend team to delete data
     integrationName: promptAPerson
     notify-email-address: test+2@transcend.io
+    prompt-a-vendor-email-settings:
```

```suggestion
    email-settings:
```

id also just call this `email-settings`, since the customer is more often than not, configuring this to be for internal usage, not vendor usage